qt/DetailsDialog.cc

```diff
@@ -386,6 +390,46 @@ void DetailsDialog::onSessionCalled(Session::Tag tag)
     }
 }
 
+void DetailsDialog::onButtonBoxClicked(QAbstractButton* button)
+{
+    if (ui_.dialogButtons->standardButton(button) == QDialogButtonBox::Close)
+    {
+        QString labels_text = ui_.labelsTextEdit->toPlainText().trimmed();
+        QString re = QStringLiteral("((,|;)\\s*)");
```

Comment:
_(nit)_ can be const (I think?)

```suggestion
        QString const labels_text = ui_.labelsTextEdit->toPlainText().trimmed();
        QString const re = QStringLiteral("((,|;)\\s*)");
```

Reply:
done